src/dotnetTestExplorer.ts

```diff
@@ -62,4 +59,13 @@ export class DotnetTestExplorer implements vscode.TreeDataProvider<vscode.TreeIt
         Executor.runInTerminal(`dotnet test --filter FullyQualifiedName~${methodName}`, this.cwd);
         AppInsightsClient.sendEvent("runTest");
     }
+
+    private evaluateWorkingDirectory(): void {

```

Comment:
I am afraid it will not fix your case, since `cd ***` will only happen when terminal is created: https://github.com/formulahendry/vscode-dotnet-test-explorer/blob/master/src/executor.ts#L7